tests/stage_2/exp-1.txt

```diff
@@ -1,2 +1,2 @@
 brun -s2 '(com (q . (list 1999 2 3))))'
-((c ("com" ((c (q (c (q (c (f 1) (c (f 1) (c (r 1) (q))))) (c (q (c (i (f (r 1)) (q 4 (q . 4) (c (f (f (r 1))) (c ((c (f 1) (c (f 1) (c (r (f (r 1))) (q))))) (q)))) (q 1)) 1)) 1))) (q 1999 2 3))) (q (26982 (c (c (q . 4) (c (c (q . 3) (c 2 (c (c (q . "function") (c 5 (q))) (c (c (q . "function") (c 11 (q))) (q))))) (q 64))) (q))) ("function" (c (q . "opt") (c (c (q . "com") (c (c (q . 1) 2) (q (29041 ("unquote" ("macros"))) (29041 ("unquote" ("symbols")))))) (q)))) ("list" ((c (q (c (f 1) (c (f 1) (c (r 1) (q))))) (c (q (c (i (f (r 1)) (q 4 (q . 4) (c (f (f (r 1))) (c ((c (f 1) (c (f 1) (c (r (f (r 1))) (q))))) (q)))) (q 1)) 1)) 1)))) ("defmacro" (c (q . "list") (c (f 1) (c (c (q . "mod") (c (f (r 1)) (c (f (r (r 1))) (q)))) (q)))))) (q)) 1))
+(a ("com" (a (q 2 (q 2 (q 2 2 (c 2 (c 3 (q)))) (c (q 2 (i 5 (q 4 (q . 4) (c 9 (c (a 2 (c 2 (c 13 (q)))) (q)))) (q 1)) 1) 1)) 1) (q 1999 2 3)) (q (26982 (c (q . 2) (c (c (q . 3) (c 2 (c (c (q . "function") (c 5 (q))) (c (c (q . "function") (c 11 (q))) (q))))) (q 64)))) ("function" (c (q . "opt") (c (c (q . "com") (c (c (q . 1) 2) (q (29041 ("unquote" ("macros"))) (29041 ("unquote" ("symbols")))))) (q)))) ("list" (a (q 2 (q 2 2 (c 2 (c 3 (q)))) (c (q 2 (i 5 (q 4 (q . 4) (c 9 (c (a 2 (c 2 (c 13 (q)))) (q)))) (q 1)) 1) 1)) 1)) ("defmacro" (c (q . "list") (c 2 (c (c (q . "mod") (c 5 (c 11 (q)))) (q)))))) (q)) 1)
```

Comment:
Are these `f/r` -> path changes an optimizer change, or are you doing these by hand?

Reply:
It must be an optimizer change, since that's the verified output, although I don't specifically remember what it is at the moment.